sdk/schemaregistry/schema-registry-avro/src/avroSerializer.ts

```diff
@@ -182,6 +183,11 @@ export class AvroSerializer<MessageT = MessageWithMetadata> {
     const entry = { id, serializer };
     this.cacheBySchemaDefinition.set(schema, entry);
     this.cacheById.set(id, serializer);
+    logger.verbose(
+      `Cache entry added or updated. Total number of entries: ${
+        this.cacheBySchemaDefinition.length
+      }; Total schema length ${(this.cacheSize += schema.length)}`
```

Comment:
What if something would fall off the cache. Would that already be reflected in the cacheSize property?

Reply:
ops, I forgot it is an LRU cache 😅

I ditched `this.cacheSize` and used the cache API to track the total length instead.